src/backend/TrafficCourts/Staff.Service/Startup.cs

```diff
@@ -35,6 +35,7 @@ public static void ConfigureApplication(this WebApplicationBuilder builder, Seri
 
         builder.AddRedis();
 
+        builder.Services.AddTransient<UserIdentityProviderHandler>();

```

Comment:
This line is already in main. Will result in a duplicate registration.

Reply:
resolved